lib/awful/widget/calendar_popup.lua

```diff
@@ -272,6 +273,7 @@ end
 -- @tparam string args.bg Wibox background color
 -- @tparam string args.font Calendar font
 -- @tparam number args.spacing Calendar spacing
+-- @tparam number args.margin Margin around calendar widget
```

Comment:
Why don't you like punctuation with docs?
In this case it matches the surrounding code, but I would really like this to be more enforced, i.e. also change it for the surrounding code (in a separate commit if you like to).